server/sonar-process/src/main/java/org/sonar/process/ProcessProperties.java

```diff
@@ -39,6 +39,7 @@
   public static final String CLUSTER_NAME = "sonar.cluster.name";
   public static final String HAZELCAST_LOG_LEVEL = "sonar.log.level.app.hazelcast";
   public static final String CLUSTER_WEB_LEADER = "sonar.cluster.web.startupLeader";
+  public static final String CLUSTER_LOCALENDPOINT = "sonar.cluster.localEndPoint";
```

Comment:
I would document that this property is internal to SonarQube, used to provide the endPoint to the Hazelcast cluster the main process created or joined and passed to the sub processes in case the new to join this hazelcast cluster